src/workflows/calls_generation.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+from typing import List
+
+import kfp
+from kfp import dsl
+import mlrun
+
+
+@kfp.dsl.pipeline()
+def pipeline(
+    amount: int,
+    generation_model: str,
+    use_small_models: bool,
+    language: str,
+    available_voices: List[str],
+    min_time: int,
+    max_time: int,
+    from_date: str,
+    to_date: str,
+    from_time: str,
+    to_time: str,
+    num_clients: int,
+    num_agents: int,
+    generate_clients_and_agents: bool = True,
+):
```

Comment:
where all the doc strings went? I'm sure I wrote them, weird 😕

Reply:
I did not see them anywhere... maybe you wrote them locally?